### ISSUE TITLE
feature-request: Add option to disable Main server downloads on frontend download page

### ISSUE THREAD

**Issue by Jackeagle** [shipperstack/shipper#229]
### Description

It would be nice to have a option to disable Main Server Downloads on frontend download page.
It should still generate API and OTA Download links from Main server so that only OTA System uses main server, while all offline downloads are linked to SourceForge Downloads :)

**Comment by ericswpark** [comment]
Blocked by #230
